osbenchmark/metrics.py

```diff
@@ -1337,6 +1343,10 @@ def __init__(self, benchmark_version, benchmark_revision, environment_name,
         self.revision = revision
         self.results = results
         self.meta_data = meta_data
+        self.latency_percentiles = None
```

Comment:
Similar to what is done for `meta_data` parameters and `results`, we can move the if statement on line 1347 to line 1325, and just have it reassigned to `latency_percentiles`.  This way, we will be keeping the section that modifies the parameters together, leveraging the `None` assigned to `latency_percentiles` in the init parameter, and keeping line 1346 identical to the other parameter formats `self.latency_percentiles = latency_percentiles`.